documentation/source/user_section/tutorials/mtr-computation/before-starting.rst

```diff
@@ -0,0 +1,15 @@
+Before starting this tutorial
+#############################
+
+1. Make sure that you have the following files in your working directory:
+
+ * ``single_subject/data/mt/mt0.nii.gz``:
+ * ``single_subject/data/mt/mt1.nii.gz``:
+ * ``single_subject/data/mt/mt1_seg.nii.gz`` : The segmented spinal cord for the MT1 image (used for registering MT0 on MT1).
+ * ``single_subject/data/mt/mask_mt1.nii.gz`` : The mask surrounding the spinal cord region of interest (used for registering MT0 on MT1).
+
+   You can get these files by downloading :sct_tutorial_data:`data_shape-metric-computation.zip`.
```

Comment:
this data seems to be missing the mt/ folder

Reply:
This too was another typo on my part. This was the wrong `.zip` -- it should say `:sct_tutorial_data:'data_mtr-computation.zip'` instead. :sweat_smile: